doc/COMPILING/COMPILING-CMAKE.md

```diff
@@ -70,9 +70,9 @@ Obtain packages specified above with your system package manager.
 		  gettext-devel
  ```
 
- This should get your environment set up to build console and tiles version of windows.
+ This should get your environment set up to build both the console and tiles versions for Windows.
```

Comment:
Isn't the console build for Windows unsupported?

Reply:
correct